lib/main.dart

```diff
@@ -0,0 +1,143 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+
+void main() {
+  runApp(MyApp());
+}
+
+class MyApp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      home: MainView(),
+    );
+  }
+}
+
+class MainView extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      backgroundColor: (Colors.blue),
+      appBar: AppBar(
+        centerTitle: true,
+        iconTheme: IconThemeData(color: Colors.black),
+        backgroundColor: Colors.yellow,
+        title: Text('To-Do List',
+            style: TextStyle(color: Colors.black, 
+            fontSize: 30)
+            ),
+        actions: [
+          IconButton(
+            icon: Icon(Icons.more_vert, 
+            color: Colors.black),
+            onPressed: () {
+              showDialog(
+                context: context,
+                builder: (BuildContext context) {
+                return AlertDialog(
+                  title: Text('All  Done  Undone'),
+
+                  //Enda funktionen jag inte riktigt lyckades implementera var drop-down-menyn för att se "All", "Done" och "Undone". Tacksam för feedback på hur man bäst gör det :)
```

Comment:
Jag tror inte det var en del av uppgiften att sortera/filtrera på "all", "done" och "undone". Från instruktionerna står det "Ni behöver heller inte kunna filtera listan". Men det vore ju fett om du kunde få in en sån funktionalitet framöver. Men designmässigt gillar jag att det kommer upp som en popup (showDialog)